### PR TITLE
Coupons table columns edited

### PR DESCRIPTION
**Pull request by MercadoJose**

### CODE REVIEW

File: src/main/java/com/backend/webproject/controller/AdminSideController.java [hunk 1]
```diff
@@ -31,8 +32,9 @@ public String showCouponsPage(Model model) {
     }
 
     @RequestMapping("/validateCoupon/{couponCode}")
-    @ResponseBody
```
Comment (alberto-2-garcia) [inline src/main/java/com/backend/webproject/controller/AdminSideController.java:34]: Why was the ResponseBody annotation erased?

File: src/main/webapp/WEB-INF/jsp/coupons.jsp [hunk 2]
```diff
@@ -156,10 +156,10 @@ prefix="c"%>
...
 												</option>
 												<option value="1">Shooter</option>
-												<option value="2">VR</option>
-												<option value="3">Strategy</option>
-												<option value="4">Simulation</option>
-												<option value="5">RPG</option>
+												<option value="2">RPG</option>
+												<option value="3">Simulation</option>
+												<option value="4">VR</option>
+												<option value="5">Strategy</option>
```
Comment (alberto-2-garcia) [inline src/main/webapp/WEB-INF/jsp/coupons.jsp:162]: Change this so the categories come from the db and aren't hardcoded.

File: src/main/webapp/WEB-INF/jsp/updateCouponDataForm.jsp [hunk 3]
```diff
@@ -161,10 +161,10 @@ prefix="c"%>
...
                                     </option>
                                     <option value="1" selected>Shooter</option>
-                                    <option value="2">VR</option>
-                                    <option value="3">Strategy</option>
-                                    <option value="4">Simulation</option>
-                                    <option value="5">RPG</option>
+                                    <option value="2">RPG</option>
+									<option value="3">Simulation</option>
+									<option value="4">VR</option>
+									<option value="5">Strategy</option>
```
Comment (alberto-2-garcia) [inline src/main/webapp/WEB-INF/jsp/updateCouponDataForm.jsp:167]: Get this from the database too.

File: src/main/webapp/WEB-INF/jsp/coupons.jsp (hunk 2 above)
Comment (alberto-2-garcia) [reply src/main/webapp/WEB-INF/jsp/coupons.jsp:162]: Well, due to time, maybe leave it like this.

File: src/main/webapp/WEB-INF/jsp/updateCouponDataForm.jsp (hunk 3 above)
Comment (alberto-2-garcia) [reply src/main/webapp/WEB-INF/jsp/updateCouponDataForm.jsp:167]: Well, due to time, maybe leave it like this.

File: src/main/java/com/backend/webproject/controller/AdminSideController.java (hunk 1 above)
Comment (MercadoJose) [reply src/main/java/com/backend/webproject/controller/AdminSideController.java:34]: My mistake, I did not noticed it was erased when fixing merge problems

Comment (alberto-2-garcia) [reply src/main/java/com/backend/webproject/controller/AdminSideController.java:34]: Ohh okok, just put it back, I think it is necesary.

Comment (MercadoJose) [reply src/main/java/com/backend/webproject/controller/AdminSideController.java:34]: I have already make the correction

**Pull request merged**